test/integration/SimplePublishConsume.test.ts

```diff
@@ -287,18 +318,20 @@ describe('Simple Publish & consume test', async () => {
       1,
       setProviderFee
     )
-    const orderId = `${orderTx.transactionHash.toLowerCase()}-${datatokenAddress.toLowerCase()}-${user1.toLowerCase()}`
-
+    await sleep(3000)
+    const orderId = `${orderTx.transactionHash.toLowerCase()}-${datatokenAddress.toLowerCase()}-${user1.toLowerCase()}-${orderTx.events.OrderStarted.logIndex.toFixed(
+      1
+    )}`
     const query = {
-      query: `query {order(id:"${orderId}"){id, providerFee, lastPriceToken{id}}}`
+      query: `query {order(id:"${orderId}"){id, providerFee, lastPriceToken{id}, eventIndex}}`
     }
 
-    await sleep(2000)
+    await sleep(3000)
     const response = await fetch(subgraphUrl, {
       method: 'POST',
       body: JSON.stringify(query)
     })
-
+    await sleep(3000)
```

Comment:
This isn't necessary. The `await` will already ensure that you get a response, the sleep doesn't change anything